pkg/node/stager.go

```diff
@@ -257,6 +268,27 @@ func readISCSITargetsFromPublishContext(pc map[string]string) []gobrick.ISCSITar
 	return targets
 }
 
+func readNVMETargetsFromPublishContext(pc map[string]string) []gobrick.NVMeTCPTargetInfo {
+	var targets []gobrick.NVMeTCPTargetInfo
+	for i := 0; ; i++ {
+		target := gobrick.NVMeTCPTargetInfo{}
+		t, tfound := pc[fmt.Sprintf("%s%d", common.PublishContextNVMETargetsPrefix, i)]
+		if tfound {
+			target.Target = t
+		}
+		p, pfound := pc[fmt.Sprintf("%s%d", common.PublishContextNVMEPortalsPrefix, i)]
+		if pfound {
+			target.Portal = p
+		}
+		if !tfound || !pfound {
+			break
```

Comment:
1. should we continue instead of break?
2. Do we need to log here?

Reply:
Just used the same way its previously used for iSCSI

Reply:
ok